habitat-lab/habitat/isaacsim/ssh_conversion.py

```diff
@@ -0,0 +1,62 @@
+"""
```

Comment:
This looks like a nice utility for your SSH-based workflow, but this doesn't need to be added to the habitat-lab repo. Most folks won't be using this workflow. This file shouldn't be part of this PR.